### ISSUE TITLE
Net power demand column

### ISSUE THREAD

**Issue by KaliPete** [pacemaker82/PredBat-Table-Card#41]
Would it be possible to make a net power demand column, where we take the predicted load, and subtract the predicted pv?

**Comment by PianSom** [comment]
Nice feature - thanks!

Does the load include power scheduled for the car? (I know - I could go and plug it in and find out, but ... :) )

**Comment by pacemaker82** [comment]
Good point, no it doesn't - nor iBoost. I will add that into the calculation on the next release!

**Comment by PianSom** [comment]
I turned on old-skool and got different colours. Is this as expected?

**Comment by pacemaker82** [comment]
Yeah I already fixed it, just not released yet. As soon as I can validate the car column impact I will release a fix.

**Comment by pacemaker82** [comment]
I've also added the change to include car and iboost into main branch, ready for release later.

**Comment by pacemaker82** [comment]
confirmed those changes and pushed new release out 1.5.7.2

**Comment by godfreym29** [comment]
> [@godfreym29](https://github.com/godfreym29) please can you post a screenshot of your original predbat plan so I can compare? There is no issues with the YAML for the card config.

![Image](https://github.com/user-attachments/assets/c92e9af4-8b3c-47fa-95dd-8988f85ae431)

**Comment by pacemaker82** [comment]
@godfreym29 got to say this is a strange one, everything is telling me that this should work - even in HTML DEBUG MODE which you have on. (I am testing here with the same).

<img width="898" alt="Image" src="https://github.com/user-attachments/assets/ce85f5ce-ccb2-447c-b7df-68a14ad49bf9" />

In the meantime, I have pushed [1.5.7.3](https://github.com/pacemaker82/PredBat-Table-Card/releases/tag/v1.5.7.3) to see if that helps, can you try it?

**Comment by godfreym29** [comment]
> [@godfreym29](https://github.com/godfreym29) got to say this is a strange one, everything is telling me that this should work - even in HTML DEBUG MODE which you have on. (I am testing here with the same).
> 


> In the meantime, I have pushed [1.5.7.3](https://github.com/pacemaker82/PredBat-Table-Card/releases/tag/v1.5.7.3) to see if that helps, can you try it?

That seems to have worked - thanks


![Image](https://github.com/user-attachments/assets/382d2b20-90e1-42f0-ad60-cb8ef85b3ef0)

**Comment by pacemaker82** [comment]
@godfreym29 great thank you. Still not sure what was wrong, but safer code fixed it.

**Comment by pacemaker82** [comment]
Closing, as delivered.
